holoviews/plotting/mpl/plot.py

```diff
@@ -240,7 +240,7 @@ def anim(self, start=0, stop=None, fps=30):
 
 
     def update(self, key):
-        if len(self) == 1 and key == 0 and not self.drawn:
+        if len(self) == 1 and ((key == 0) or (key == self.keys[0])) and not self.drawn:
```

Comment:
Is the idea that ``self.keys[0]`` here normally matches ``init_key``?

Reply:
Correct usually plots are initialized to the first frame.